Post message target handling in Luigi Client (#651)
SAP/luigi: README.md

- Make the micro frontend accessible only through HTTPS.
- Add Content Security Policies (CSPs).
- Make the Access-Control-Allow-Origin HTTP header as restrictive as possible.

> - Maintain a whitelist with trusted domains and compare it with the origin of luigi core app. The origin will be passed when you call the init listener in your micro frontend. Stop further processing in case of a non-matching origin.

Reviewer: ```suggestion
- Maintain a whitelist with trusted domains and compare it with the origin of Luigi core application. The origin will be passed when you call the init listener in your micro frontend. Stop further processing if the origin does not match.
```
